src/main/java/io/stargate/sgv2/jsonapi/api/model/command/table/IndexDesc.java

```diff
@@ -12,15 +12,19 @@
  * <p>Does not include the command options, because we don't want them in the list indexes command.
  * Needs @JsonProperty annotation because it is not using bean accessor methods.
  */
-@JsonPropertyOrder({TableDescConstants.IndexDesc.NAME, TableDescConstants.IndexDesc.DEFINITION})
+@JsonPropertyOrder({
```

Comment:
This is only needed if we serialize type -- not used on deserialization.
So could perhaps be removed?

Reply:
This is the interface used to describe indexes by the `listIndexes` command